plugin/vim_fourmolu.vim

```diff
@@ -11,6 +11,13 @@ if !exists("g:fourmolu_executable") && executable("fourmolu")
     let g:fourmolu_executable = "fourmolu"
 endif
 
+" If executable contains 'fourmolu' (case insensitve), add -q flag
+if !exists("g:fourmolu_quiet") && g:fourmolu_executable =~? "fourmolu"
+    let g:fourmolu_quiet = " -q "
+else
+    let g:fourmolu_quiet = ""
```

Comment:
Not sure if there is a better way of doing this and also checking `!exists("g:fourmolu_quiet")`, but it works.